integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxRepairTests.java

```diff
@@ -155,6 +155,8 @@ public void testSDXMediumDutyRepair(TestContext testContext) {
                     return testDto;
                 })
                 .awaitForHostGroups(List.of("gateway", "idbroker"), InstanceStatus.DELETED_ON_PROVIDER_SIDE)
+                .await(SdxClusterStatusResponse.CLUSTER_UNREACHABLE, Set.of(SdxClusterStatusResponse.NODE_FAILURE),
```

Comment:
Is it the only place where we have this kind of flakiness?

Reply:
good catch, thanks, I added it to one more testcase

Reply:
You didn't found other test case?

Reply:
yes, it was in the same class